driver-core/src/test/resources/retryable-reads/changeStreams-client.watch-serverErrors.json

```diff
@@ -13,7 +13,7 @@
       ]
     }
   ],
-  "database_name": "admin",
+  "database_name": "retryable-reads-tests",
```

Comment:
I am not clear on why the retryable reads test changes had to be pulled into this PR.  Is it so we would have consistent fullDocument fields across all the tests?